torchmetrics/functional/regression/mean_squared_error.py

```diff
@@ -20,7 +20,8 @@
 
 def _mean_squared_error_update(preds: torch.Tensor, target: torch.Tensor) -> Tuple[torch.Tensor, int]:
     _check_same_shape(preds, target)
-    sum_squared_error = torch.sum(torch.pow(preds - target, 2))
+    diff = preds - target
+    sum_squared_error = torch.sum(diff * diff)
```

Comment:
Do you know if there is a speed impact between diff * diff and torch.pow ?

Reply:
diff * diff is approximately twice as fast compared to torch.pow (with a little variation depending on the size of the input).

Reply:
That's only true for torch.pow(..., 2). For higher powers, the pow function is considererd to be faster